master/buildbot/test/unit/test_process_builder.py

```diff
@@ -261,6 +261,7 @@ def canStartBuild(bldr, slave, breq):
         def canStartBuild_deferred(bldr, slave, breq):
             record.append((bldr, slave, breq))
             return (slave, breq) == ('slave', 100)
+            # XXX(sa2ajj): too many returns?
             return defer.succeed((slave, breq) == ('slave', 100))
```

Comment:
I suspect I (or whoever wrote this) was trying to test both deferred and non-deferred versions of `canStartBuild`.  It's probably fine to delete the first of the two `return` lines.